apollo-router/src/plugins/telemetry/metrics/apollo.rs

```diff
@@ -94,7 +94,7 @@ impl Config {
         schema_id: &str,
         batch_processor: &BatchProcessorConfig,
     ) -> Result<MetricsBuilder, BoxError> {
-        tracing::debug!("creating otlp metrics exporter");
+        tracing::debug!(endpoint = %endpoint, "creating Apollo OTLP metrics exporter");
```

Comment:
NIT:

`endpoint` == `endpoint = %endpoint`. Usually you use the second syntax if you are renaming a variable.

e.g.: `something = %endpoint`, would take the DISPLAY value of endpoint and record it as an attribute with a name of `something`.

Reply:
That's weird...